pt-BR/active_model_basics.md

```diff
@@ -1,35 +1,33 @@
 **NÃO LEIA ESTE ARQUIVO NO GITHUB, OS GUIAS SÃO PUBLICADOS NO https://guiarails.com.br.**
 **DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON https://guides.rubyonrails.org.**
 
-Active Model Basics
+Basico de Active Model
```

Comment:
```suggestion
Básico de Active Model
```

Reply:
Acredito que também podemos colocar Active Model como itálico, faz sentido @campuscode/guia-rails-team ?

Reply:
> Acredito que também podemos colocar Active Model como itálico, faz sentido @campuscode/guia-rails-team ?

Sim, faz sentido. Acho que podemos pensar em criar um `style guide` para as traduções, para facilitar na padronização.

Reply:
Acho que esse ficou faltando @LuizaMagalhaes 

Sobre o style guide a @erikacamposdesign e o @akaninja estão trabalhando nisso, mas deve levar um tempinho :disappointed:

Reply:
opa 😓  agora foi